src/js/server/standaloneRenderer.js

```diff
@@ -0,0 +1,119 @@
+// TODO:  If we find ourselves using this component more often than in just mailing, move it to `gpii-handlebars`.
+//
+// A standalone handlebars renderer designed for use in rendering mail templates.
+//
+// Although this does not itself require express, it can work with any helper functions that extend the
+// `gpii.express.helper` grade, but which do not themselves require express.
+//
+// The directory conventions used with express are partially supported, as follows:
+//
+//  1. Any templates in the `partials` subdirectory relative to `options.templateDir` will be registered as partials for use in `{{>partial}}` statements.
+//  2. All other templates are expected to be stored in a `pages` subdirectory relative to `options.templateDir`.
+//
+"use strict";
+var fluid = fluid || require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+fluid.registerNamespace("gpii.handlebars.standaloneRenderer");
+
+var Handlebars = require("handlebars");
```

Comment:
Anomalous use of capital letter for import we don't plan to use as a constructor